gapic/api/artman_logging.yaml

```diff
@@ -9,8 +9,10 @@ common:
   output_dir: ${REPOROOT}/artman/output
   gapic_api_yaml:
     - ${REPOROOT}/googleapis/google/logging/v2/logging_gapic.yaml
+  enable_batch_generation: True
 java:
   final_repo_dir: ${REPOROOT}/gcloud-java/gcloud-java-logging
+  batch_dir_name: gcloud-java/gcloud-java-logging
```

Comment:
Hmmm. Now that you bring up those points and I think more about it, I am starting to think maybe the staging repo **shouldn't** mirror the google cloud repos. Adding the mirroring seems like it introduces a lot of new settings once we start thinking about non-GCP cases, staging EAP cases, and the existing single generation case, and it also ties the staging repo to the google cloud repos in an annoying way. Case in point - gcloud-java just changed to google-cloud-java. I think we may be better off having the staging repo just use `<lang>/<api>` variables like it was doing, and introduce a setting to point to the relevant google cloud repo when we add a publish step that pulls from the staging repo.